.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -164,7 +164,7 @@ jobs:
         id: cilium-config
         uses: ./.github/actions/cilium-config
         with:
-          image-tag: ${{ steps.vars.outputs.sha }}
+          image-tag: ${{ steps.vars.outputs.sha }}-bootid-override
```

Comment:
This implementation of per-kind-node bootid is actually per-agent-process bootid, it changes after agent restarts (upgrade). ci-ipsec-upgrade goes red due to that.

I'll add a hidden flag since everyone is okay with it. Hidden flag solution is more low-carbon (don't need to run a new ci build job) and it can be real per-kind-node bootid.